torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -407,6 +407,10 @@ def _parallelogram_to_bounding_boxes(parallelogram: torch.Tensor) -> torch.Tenso
         torch.int32,
         torch.int64,
     )
+    if int_dtype:
+        # Does not apply the transformation to `int` boxes as the rounding error
+        # will typically not ensure the resulting box has a rectangular shape.
+        return parallelogram.clone()
```

Comment:
Is it better to return the parallelogram as-is, i.e. *really* not a rectangle, or still try to do the conversion and return something that is closer to a rectangle?

Separately it makes me wonder, maybe we should completely prevent rotated bounding boxes of integer dtype? That would probably make our life a lot easier?